app/models/attendance.rb

```diff
@@ -0,0 +1,3 @@
+class Attendance < ApplicationRecord
+    belongs_to :employee
+end
```

Comment:
Blank line at the end here

Reply:
Agree with German 
You should activate this [setting](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files) on vs code